ui/ember-cli-build.js

```diff
@@ -35,6 +35,14 @@ module.exports = function(defaults) {
       grid: true,
       browsers: ['defaults', 'ie 11'],
     },
+    autoImport: {
+      webpack: {
+        // this makes `unsafe-eval` CSP unnecessary
+        // see https://github.com/ef4/ember-auto-import/issues/50
+        // and https://github.com/webpack/webpack/issues/5627
+        devtool: 'inline-source-map',
```

Comment:
@johncowen pointed out that this looks like sourcemaps are being included all of the time - I tested to make sure though, the prod build of the assets is the same with and without this setting:

```
File sizes:
 - ../pkg/web_ui/assets/auto-import-fastboot-d41d8cd98f00b204e9800998ecf8427e.js: 0 B
 - ../pkg/web_ui/assets/vault-015a748570eb6618e787e354bb62ccd4.css: 387.54 KB (44.15 KB gzipped)
 - ../pkg/web_ui/assets/vault-c80e56e2badf9084e8b77d18a2ef5d66.js: 1.08 MB (159.22 KB gzipped)
 - ../pkg/web_ui/assets/vendor-265ca8cddb2e5db77ff28daec5601ced.js: 1.21 MB (332.36 KB gzipped)
 - ../pkg/web_ui/assets/vendor-9ea7d400c0cec7682e8871df14073823.css: 7.89 KB (2.54 KB gzipped)
 - ../pkg/web_ui/ember-fetch/fastboot-fetch-027a9422b3692a30fce1385c8b1de3a6.js: 299 B (205 B gzipped)
```